packages/rx_bloc_cli/lib/src/processors/android/android_build_gradle_processor.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:rx_bloc_cli/src/extensions/string_buffer_extensions.dart';
+
+import '../common/abstract_processors.dart';
+
+/// String processor used for processing android/build.gradle file
+class AndroidBuildGradleProcessor extends StringProcessor {
+  /// Android build gradle processor constructor
+  AndroidBuildGradleProcessor(super.args);
+
+  @override
+  String execute() {
+    if (input == null) return '';
+    final buffer = StringBuffer(input!);
+
+    if (args.analyticsEnabled) {
+      _applyAnalyticsOptions(buffer);
+    }
+
+    return buffer.toString();
+  }
+
+  void _applyAnalyticsOptions(StringBuffer buffer) {
+    final (start, end) = buffer.getGradleSectionLastLineRange('dependencies');
+    final content = '''
+\n        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.9.9'
```

Comment:
do we really need to hardcode the version ?

Reply:
Yes, it is required for analytics.

Also, omitting a version results in build errors.

![Screenshot 2024-02-07 at 20 58 22](https://github.com/Prime-Holding/rx_bloc/assets/61236867/43880ddd-9a16-4469-af9c-1849c763b2eb)